lib/api/dsl/methods.js

```diff
@@ -8,8 +8,231 @@ var
   util = require('util'),
   geohash = require('ngeohash'),
   units = require('node-units'),
-  methods,
-  geoUtil = {};
+  methods;
+
+var geoUtil = {
```

Comment:
Can we move the `geoUtil` object below all methods ?  Or maybe like you suggest, in other file